utils.py

```diff
@@ -0,0 +1,34 @@
+# autopep8: off
+import os
+import jwt
+import json
+import my_settings
+
+from pathlib                import Path
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.exceptions import ImproperlyConfigured
+from user.models            import User
+
+
+def signin_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token     = request.headers.get('Authorization', None)
+            key       = my_settings.SECRET.get('JWT_KEY')
+            algorithm = my_settings.SECRET.get('JWT_ALGORITHM')
+
+            if token:
+                decode       = jwt.decode(token, key, algorithm=algorithm)
```

Comment:
```suggestion
                decode       = jwt.decode(token, key, algorithms=algorithm)
```
https://pyjwt.readthedocs.io/en/stable/#example-usage
참고해주세요

Reply:
넵! 감사합니다